src/test/java/org/esbtools/gateway/resync/ResyncErrorMessagesTest.java

```diff
@@ -0,0 +1,49 @@
+package org.esbtools.gateway.resync;
+
+import org.esbtools.gateway.resync.service.ResyncErrorMessages;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class ResyncErrorMessagesTest {
+    ResyncRequest resyncRequest;
+
+    @Before
+    public void setUp() {
+        resyncRequest = new ResyncRequest();
+        resyncRequest.setEntity("entity");
+        resyncRequest.setSystem("system");
+        resyncRequest.setKey("key");
+        resyncRequest.setValues(Arrays.asList("value1","value2"));
+    }
+
+    @After
+    public void tearDown() {
+        resyncRequest = null;
+    }
+
+    @Test
+    public void testInvalidSystem() throws Exception {
+        Assert.assertEquals(ResyncErrorMessages.INVALID_SYSTEM + "system", ResyncErrorMessages.invalidSystem("system"));
+    }
+
+    @Test
+    public void testIncompleteRequest() throws Exception {
+        assertEquals(ResyncErrorMessages.INCOMPLETE_REQUEST + resyncRequest, ResyncErrorMessages.incompleteRequest(resyncRequest));
+    }
+
+    @Test
+    public void testSystemNotConfigured() throws Exception {
+        assertEquals(ResyncErrorMessages.SYSTEM_NOT_CONFIGURED + "system", ResyncErrorMessages.systemNotConfigured("system"));
+    }
+
+    @Test
+    public void testResyncFailed() throws Exception {
+        assertEquals(ResyncErrorMessages.RESYNC_FAILED + resyncRequest, ResyncErrorMessages.resyncFailed(resyncRequest));
```

Comment:
In a way, (at least I sometimes think of it this way) that's what tests sometimes are... another place to say the same logic in a different way :). But, I think keeping the tests really simple in that way is also fine, so sounds good to me. We could make the constants package private if it turns out we don't need them visible for any other reason than testing.